lib/.eslintrc

```diff
@@ -1,158 +1,9 @@
 {
```

Comment:
This needed to be split up into two files: the base file which is effectively exactly what this used to be, and then this file which extends the base and ads the a11y linting to it.  This was done so that the unit test linting configuration can extend the base file and thereby avoid a11y linting, which IMO would just get in the way for unit tests.